app/audit/models/ForecastTaxCalculationAuditModel.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package audit.models
+
+import audit.Utilities.userAuditDetails
+import auth.MtdItUserWithNino
+import implicits.ImplicitDateParser
+import models.liabilitycalculation.EndOfYearEstimate
+import play.api.libs.json.{JsObject, JsValue, Json}
+import utils.Utilities._
+
+
+case class ForecastTaxCalculationAuditModel(user: MtdItUserWithNino[_], endOfYearEstimate: EndOfYearEstimate)
+  extends ExtendedAuditModel with ImplicitDateParser {
+
+  override val transactionName: String = enums.TransactionName.ForecastTaxCalculation
+  override val auditType: String = enums.AuditType.ForecastTaxCalculation
+
+  private val totalEstimatedIncome: Option[Int] = endOfYearEstimate.totalEstimatedIncome
```

Comment:
not sure if we should use Option[Int] here to represent amount

Reply:
clarification required from the BAs level regarding format for each amount we store in this event; I've asked @Olga to clarify requirements for this story;